integration_tests/pages/page.ts

```diff
@@ -66,6 +71,24 @@ export default abstract class Page {
     })
   }
 
+  shouldShowPrisonApplications(applications: Array<Cas2v2ApplicationSummary>): void {
+    applications.forEach(application => {
+      const { personName, nomsNumber, createdByUserName } = application
+      const statusLabel = application.latestStatusUpdate?.label
+
+      cy.contains(personName)
+        .should('have.attr', 'href', paths.applications.overview({ id: application.id }))
+        .parent()
+        .parent()
+        .within(() => {
+          cy.get('th').eq(0).contains(personName)
+          cy.get('td').eq(0).should('contain.text', nomsNumber)
+          cy.get('td').eq(1).should('contain.text', createdByUserName)
+          cy.get('td').eq(2).should('contain.text', statusLabel)
+        })
+    })
+  }
+
```

Comment:
sorry for the late comment but would this no be better off living on the prison applications page class?